open_instruct/grpo_fast.py

```diff
@@ -1932,7 +1915,7 @@ def create_model_and_optimizer(
             else:
                 raise ValueError(f"Unknown tool: {tool}")
 
-    actor_manager = vllm_utils3.ActorManager.remote()
+    actor_manager = ray.remote(vllm_utils3.ActorManager).remote()
```

Comment:
Is this change necessary since ActorManager is already decorated with ray.remote?